42364 Add ability to launch update from update notification icon insi…

tyndmar opened the pull request:
…de ASI

Program changed: system/mainMenu.w
Changed launch target when new version available and user clicks icon

Commit:
42364 Add ability to launch update from update notification icon inside ASI
Program changed: system/mainMenu.w
Changed launch target when new version available and user clicks icon

brad-advantzware reviewed (changes requested):
@mark-advantzware  I don't understand this.  Why did we switch from the NK1 to hard-coding the path?  If we do this, we need to change the existing NK1.  But then the code still requires the description to be non-blank?

Commit:
system/MainMenu.w
- Modify version check - this is the only location for this code INSIDE ASI, so left in place rather that develop separate include
sysconst.i
- change version for testing only

Commit:
42364 Add ability to launch update from main menu
Programs changed:
asiUpdate.w
asiUpdateDB.w
asiUpdateEnv.w
continuing mods to support new download strategy
not complete, but need commit for testing

brad-advantzware reviewed (changes requested):
@mark-advantzware sysconst.i  - changed to 16.8.4?

Commit:
Delete sysconst.i
